docs/src/app/shared/navigation-focus/navigation-focus.spec.ts

```diff
@@ -123,13 +121,14 @@ describe('Navigation focus service', () => {
     <button id="target2">Target 2</button>
     <button class="no-id" focusOnNavigation>Target 3</button>
   `,
-  standalone: false,
+  standalone: true,
```

Comment:
`standalone: true` should be the default so we should just be able to remove this property entirely right?